### DISCUSSION TITLE
Google maps bookmarks

### DISCUSSION THREAD

**Discussion by Orejane** [organicmaps/organicmaps#1074] [Q&A]
Hi, is it possible to import Google maps bookmarks to organicmaps ?

**Comment by biodranik** [comment]
Several solutions are described here:

https://webapps.stackexchange.com/questions/16177/is-there-a-way-to-export-my-starred-locations-from-google-maps

**Comment by rtsisyk** [comment]
I research this topic. There is no working solution other than Google Takeout. 

1. Download Google Maps bookmarks from https://takeout.google.com/ in GeoJSON format.
2. Use https://umap.openstreetmap.fr to work with your bookmarks online and convert to KML.
3. Import KML into OrganicMaps.

**Reply by biodranik** [reply]
Is GeoJSON a popular format? Can/should we add its support?

**Reply by rtsisyk** [reply]
I don't know how popular or unpopular GeoJSON, but Google Takeout uses this format and it would be nice to support it in Organic Maps.
